### PR TITLE
Flow coverage

### PR DESCRIPTION
**Pull request by ycleptkellan**
First pass at covering the `Flow` component.

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -12,5 +12,5 @@ module.exports = {
     },
     collectCoverageFrom: ['src/**/*.{ts,tsx}'],
     coveragePathIgnorePatterns: ['/*.d.ts$', 'lambda-src', 'webpack', 'testUtils', '__test__.ts'],
-    setupFiles: ['<rootDir>/rafShim.js', '<rootDir>/enzymeAdapter.js']
+    setupFiles: ['<rootDir>/setup.js']
```
Comment (ycleptkellan) [inline jest.config.js]: figured we could just consolidate these

File: package.json [hunk 2]
```diff
@@ -90,7 +90,7 @@
     "css-loader": "0.26.4",
     "enzyme": "3.3.0",
     "enzyme-adapter-react-16": "1.1.1",
-    "enzyme-to-json": "3.1.4",
+    "enzyme-to-json": "3.3.3",
```
Comment (ycleptkellan) [inline package.json]: `React.Fragment` support for snapshots 🎉

File: src/store/helpers.ts [hunk 3]
```diff
@@ -227,3 +227,41 @@ export const getGhostNode = (fromNode: RenderNode, nodes: RenderNodeMap) => {
 
     return ghostNode;
 };
+
+export const getRenderNodeMap = ({ nodes, _ui }: FlowDefinition) => {
```
Comment (ycleptkellan) [inline src/store/helpers.ts]: needed to be able to get a `RenderNodeMap` in `Flow.test.ts`

File: src/component/Flow.test.ts [hunk 4]
```diff
@@ -0,0 +1,550 @@
...
+import { getActivity } from '../external';
+import { FlowDefinition } from '../flowTypes';
+import ActivityManager from '../services/ActivityManager';
+import Plumber from '../services/Plumber';
+import { ConnectionEvent, createStore, initialState } from '../store';
+import { getCollisions, getGhostNode, getRenderNodeMap } from '../store/helpers';
+import { createSetup, createSpy, getSpecWrapper } from '../testUtils';
+import { getBaseLanguage } from '../utils';
+
+jest.mock('../services/ActivityManager');
```
Comment (ycleptkellan) [inline src/component/Flow.test.ts]: auto-mocking `Activity` and `Plumber` classes here

File: src/component/Flow.tsx [hunk 5]
```diff
@@ -323,49 +325,39 @@ export class Flow extends React.Component<FlowStoreProps> {
...
-        const dragNode = this.getDragNode();
-        const nodes = this.getNodes();
-        const simulator = this.getSimulator();
-
         return (
-            <div key={this.props.definition.uuid}>
-                {simulator}
-                {dragNode}
-                {nodeEditor}
+            <>
```
Comment (ericnewcomer) [inline src/component/Flow.tsx]: whoa, weird. is this a fragment?

**Comment by ycleptkellan** [comment]
Aye, moved those flows to a `__test__` directory to start. will look at these tests, others and think on this, get your feedback 👍

**Pull request merged**